db/migrate/20240326003655_create_tags.rb

```diff
@@ -0,0 +1,13 @@
+class CreateTags < ActiveRecord::Migration[7.1]
+  def change
+    create_table :tags do |t|
+      t.text :name
+
+      t.timestamps
+    end
+
+    ["Abstract", "Representational", "Figurative", "Non-objective",
+     "Geometric", "Site-specific", "Immersive", "Environmental",
+     "Conceptual", "Social practice", "Portrait", "Landscape"].each { | tag_name | Tag.create!(name: tag_name) }
+  end
```

Comment:
I initially added this to `seeds.rb` file but running the seed command it also added the fixtures to the database so I thought we don't want to seed staging and other dev environments so the migration is a better place to add these.

Reply:
Yeah, we won't run `db:seed` ever in production.